env.bzl

```diff
@@ -13,6 +13,18 @@ py_runtime(
 )
 """
 
+def _label_from_condabin(rctx, exe_name):
+    return Label("@{}//:{}/condabin/{}{}".format(rctx.attr.conda_repo,
+        rctx.attr.conda_dir,
+        exe_name,
+        CONDA_EXT_MAP[get_os(rctx)]))
+
+def _user_chosen_executable(rctx, use_mamba):
+    if use_mamba:
+        return _label_from_condabin(rctx, "mamba")
+    return str(rctx.path(_label_from_condabin(rctx, "conda")))
```

Comment:
As we are passing `rctx` along let's just get `use_mamba` from `rctx.attr.use_mamba`